invenio_app_ils/config.py

```diff
@@ -678,12 +678,12 @@ def _(x):
         record_class=Vocabulary,
         record_serializers={
             "application/json": (
-                "invenio_records_rest.serializers" ":json_v1_response"
+                "invenio_app_ils.records.serializers:json_v1_response"
```

Comment:
@equadon I had to change the serializers otherwise it was not working, 500 when fetching the vocabularies... was it working with you with default serializers?
Python 3 in your venv?

Reply:
Hm strange. Yes, everything was working when I was running it but this change seems good.